frontend/packages/console-plugin-sdk/src/typings/storage-class-params.ts

```diff
@@ -0,0 +1,16 @@
+import { Extension } from '@console/plugin-sdk';
+
+namespace ExtensionProperties {
+  export interface StorageClassProvisioner {
+    getStorageClassProvisioner: () => any;
```

Comment:
We should have a better type than any. it looks like a custom obejct with reucrring fields. Maybe something like
```
{
[key: string] : {
name: string
hiintText: string;
value: string;
visible: () => boolean;
}
}
```